libraries/helper.rb

```diff
@@ -51,16 +55,52 @@ def omnibus_toolchain_enabled?
       !windows?
     end
 
-    def build_user_shell
-      if omnibus_toolchain_enabled?
-        "/opt/#{node['omnibus']['toolchain_name']}/embedded/bin/bash"
+    def windows_safe_path_expand(arg)
+      ::File.expand_path(arg).gsub(::File::SEPARATOR, ::File::ALT_SEPARATOR || File::SEPARATOR)
+    end
+  end
+end
+
+module Omnibus
+  # Recipe Helpers
+  module ResourceHelpers
```

Comment:
What purpose does moving private helper methods for the `omnibus_build` resource out into a separate file/module? Please leave all methods with the resource and save the indirection.